test/services/role/services/userPermissions.test.js

```diff
@@ -0,0 +1,147 @@
+const { expect } = require('chai');
+const app = require('../../../../src/app');
+const testObjects = require('../../helpers/testObjects')(app);
+
+const userPermissions = app.service('/permissions/user');
+
+describe('userPermissions', async () => {
+	let server;
+	const ROLES = {
+		TEST: 'test',
+		OTHER: 'other',
+		EXTENDED: 'extended',
+		MULTIPLE: 'multiple',
+		XX: 'xx',
+		NOTHING: 'nothing spezial',
+	};
+	const testPermissions = [
+		'SINGING',
+		'DANCE_RAIN',
+		'WALK_LINES',
+		'RUN_FLOOR',
+	];
+
+	const otherPermissions = [
+		'SITTING',
+		'SITTING_ON_CHAIR',
+		'SITTING_ON_DESK',
+	];
+
+	let testRole;
+
+	let testRoleWithDiffrentPermissons;
+
+	let otherRole;
+
+	let testSchool;
+
+	let testSchool2;
+
+	let testUser;
+
+	let testUser2;
+
+	let otherUser;
+
+	let accountTestUser;
+	let accountTestUser2;
```

Comment:
Codacy found an issue: ['accountTestUser2' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5786177)